cmd/iam.go

```diff
@@ -776,6 +802,15 @@ func (sys *IAMSys) SetGroupStatus(group string, enabled bool) error {
 
 // GetGroupDescription - builds up group description
 func (sys *IAMSys) GetGroupDescription(group string) (gd madmin.GroupDesc, err error) {
+	ps, err := sys.PolicyDBGet(group, true)
+	if err != nil {
+		return gd, err
+	}
+	policy := ""
+	if len(ps) > 0 {
+		policy = ps[0]
```

Comment:
why only the first one? a comment will help.

Reply:
because a group has at most one policy. added comment.